### DISCUSSION TITLE
Source map for generated .css file

### DISCUSSION THREAD

**Discussion by ikrbasak** [tailwindlabs/tailwindcss#8718] [Help] [answered]
I am using TailWind CSS CLI as the following
```
npx tailwindcss -i ./dev/input.css -o ./src/output.css --watch --minify
```
Though this generates a minified .css file, it does not create any source map for that.
**How can the source map be generated?**

Thank you.

**Comment by CFarhad** [comment, marked as answer]
Here is your answer [#55](https://github.com/tailwindlabs/tailwindcss-setup-examples/issues/55)
If it wasn't helpful, I suggest using postcss
